src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_CompoundAssignmentOperator.cs

```diff
@@ -403,29 +403,32 @@ private BoundIndexerAccess TransformIndexerAccess(BoundIndexerAccess indexerAcce
 
         private BoundExpression TransformImplicitIndexerAccess(
             BoundImplicitIndexerAccess indexerAccess,
-            bool isRegularCompoundAssignment,
+            bool isRegularAssignmentOrRegularCompoundAssignment,
+            bool inInitializer,
```

Comment:
>inInitializer [](http://example.com/codeflow?start=17&length=13)

Is this specifically about an object initializer? Consider making the name more specific if that is the case